DQM/SiStripMonitorSummary/src/SiStripClassToMonitorCondData.cc

```diff
@@ -51,15 +51,17 @@
 SiStripClassToMonitorCondData::SiStripClassToMonitorCondData(edm::ParameterSet const& iConfig,
                                                              edm::ConsumesCollector iC)
     : conf_(iConfig) {
-  monitorPedestals_ = iConfig.getParameter<bool>("MonitorSiStripPedestal");
-  monitorNoises_ = iConfig.getParameter<bool>("MonitorSiStripNoise");
-  monitorQuality_ = iConfig.getParameter<bool>("MonitorSiStripQuality");
-  monitorApvGains_ = iConfig.getParameter<bool>("MonitorSiStripApvGain");
-  monitorLorentzAngle_ = iConfig.getParameter<bool>("MonitorSiStripLorentzAngle");
-  monitorBackPlaneCorrection_ = iConfig.getParameter<bool>("MonitorSiStripBackPlaneCorrection");
-  monitorLowThreshold_ = iConfig.getParameter<bool>("MonitorSiStripLowThreshold");
-  monitorHighThreshold_ = iConfig.getParameter<bool>("MonitorSiStripHighThreshold");
-  monitorCabling_ = iConfig.getParameter<bool>("MonitorSiStripCabling");
+  const auto& mPSet = iConfig.getParameter<edm::ParameterSet>("MonitorSiStrip_PSet");
```

Comment:
Sorry to stir the pot more, but I just want to point out that if you would do
```py
SiStripAnalyserHI = cms.EDAnalyzer("SiStripAnalyser",
    OnDemandMonitoring.MonitorSiStrip_PSet,
    StaticUpdateFrequency    = cms.untracked.int32(1),
```
instead of current
```py
SiStripAnalyserHI = cms.EDAnalyzer("SiStripAnalyser",
    StaticUpdateFrequency    = cms.untracked.int32(1),
    ....
    MonitorSiStrip_PSet          = OnDemandMonitoring.MonitorSiStrip_PSet.clone(...)
```
these changes in the configuration reading side would not be needed.

On the other hand, there is already another PSet, `FillConditions_PSet`, so the change is not dramatically different from the existing content (and there may be other motivations for it).

Reply:
Thanks @makortel 
I decided the current solution for the sake of not mixing the ordering of the parameters, which are many. So unless you have a strong opinion I would leave as it is

Reply:
No strong feelings, just wanted to tell what is possible to do.